finalPrep/annotations.js

```diff
@@ -3,40 +3,94 @@
     clothes: clothes,
+  //the clothes key is assigned the value clothes, which is a parameter for buildABear
     exterior: fur,
+  //the exterior key is assigned the value fur, which is a parameter for buildABear
     cost: 49.99,
+  //the cost key is assigned the static value of an integer with a float of 49.99
     sayings: [greeting, powerSaying, "Goodnight my friend!"],
-    isCuddly: true,
+  //the sayings is assigned an array which takes 2 variables previously assigned to strings and the static string "Goodnight my friend!"
+    isCuddly: true
+  //isCuddly is assigned to a static boolean value of true
   };
 
-  return builtBear
+  return builtBear;
+//returns builtBear, a completed bear object after buildABear was called passing in the arguments to the parameters
+//I validated this function by assigning a variable to what the function returned. Example follows:
+/*
+var myBear = buildABear('Fluffy', 4, 'brown', ['pants', 'jorts', 'tanktop'], 'give you nightmares');
+console.log(myBear);
+*/
 }
 
 buildABear('Fluffy', 4, 'brown', ['pants', 'jorts', 'tanktop'], 'give you nightmares');
-buildABear('Sleepy', 2, 'purple', ['pajamas', 'sleeping cap'], 'sleeping in');
+//function buildABear is called passing in the string value 'Fluffy' for name parameter, the integer 4 for the age parameter, the string 'brown' for the fur parameter
+//the array of ['pants', 'jorts', 'tanktop'] for the clothes parameter and the string 'give you nightmares' for the specialPower parameter
+/*
+{basicInfo: [Fluffy, 4],
+clothes: ['pants', 'jorts', 'tanktop'],
+exterior: brown,
+cost: 49.99,
+sayings: ['Hey partner! My name is Fluffy - will you be my friend?', 'Did you know I can give you nightmares?', 'Goodnight my friend!'],
+isCuddly: true}
+*/
 
+buildABear('Sleepy', 2, 'purple', ['pajamas', 'sleeping cap'], 'sleeping in');
+//calls the function buildABear passing in the string value 'Sleepy' for the name parameter, the integer 2 for the age parameter, the string 'purple' for the fur parameter
+//the array of ['pajamas', 'sleeping cap'] for the clothes parameter and the string 'sleeping in' for the specialPower parameter.
+/*
+{basicInfo: ['Sleepy', 2],
+clothes: ['pajamas', 'sleeping cap'],
+exterior: purple,
+cost: 49.99,
+sayings: ['Hey partner! My name is Sleepy - will you be my friend?', 'Did you know that I can sleeping in?', 'Goodnight my friend!'],
+isCuddly: true}
+*/
 
 
-//FizzBuzz
+//
+// //FizzBuzz
 function fizzBuzz(num1, num2, range) {
+//function fizzBuzz is created taking three parameters that are integers
   for (var i = 0; i <= range; i++) {
+  //a loop starting the variable i at 0 and continuing to loop as long as i is less than or equal to the range and adding 1 to i each time it is run
     if (i % num1 === 0 && i % num2 === 0) {
+    //if statement first checks if the variable i is divided evenly(has a remainder of 0) by the first argument AND it also divides evenly by the second argument....
```

Comment:
Nice explanations of loops and of a modulo. Looking at code line by line helps find areas to research.